autowiring/AutoPacket.h

```diff
@@ -332,12 +332,12 @@ class AutoPacket:
       std::lock_guard<std::mutex> lk(m_lock);
       auto& entry = m_decorations[key];
       entry.SetKey(key); // Ensure correct type if instantiated here
-      if(entry.satisfied ||
-         entry.isCheckedOut)
+      if(entry.m_state == DispositionState::PartlySatisfied ||
```

Comment:
I see why you need `PartlySatisfied`, now.  I guess there's not really any avoiding it, in order to prevent ambiguity.